Model/Api/Debug.php

```diff
@@ -57,28 +59,33 @@ class Debug implements DebugInterface
 	 */
 	private $configHelper;
 
+	/**
+	 * @var ModuleRetriever
+	 */
+	private $moduleRetriever;
+
 	/**
 	 * @param DebugInfoFactory $debugInfoFactory
-	 * @param PluginVersionFactory $pluginVersionFactory
 	 * @param StoreManagerInterface $storeManager
 	 * @param HookHelper $hookHelper
 	 * @param ProductMetadataInterface $productMetadata
 	 * @param ConfigHelper $configHelper
+	 * @param ModuleRetriever $moduleRetriever
 	 */
 	public function __construct(
 		DebugInfoFactory $debugInfoFactory,
-		PluginVersionFactory $pluginVersionFactory,
 		StoreManagerInterface $storeManager,
```

Comment:
@Jiyuan-Bolt Since you removed $pluginVersionFactory in the __construct method, should we should remove this line? See here: http://prntscr.com/rjt3ou

Reply:
good catch, fixed @johnmccombs1